routers/web/auth/webauthn.go

```diff
@@ -45,24 +45,24 @@ func WebAuthn(ctx *context.Context) {
 // WebAuthnLoginAssertion submits a WebAuthn challenge to the browser
 func WebAuthnLoginAssertion(ctx *context.Context) {
 	// Ensure user is in a WebAuthn session.
-	idSess := ctx.Session.Get("twofaUid")
-	if idSess == nil {
+	idSess, ok := ctx.Session.Get("twofaUid").(int64)
```

Comment:
It will panic if there is no session named `twofaUid`.

Reply:
https://goplay.space/#-2Z8pKmnVda

Reply:
The use of the `x, ok := something.(otherthing)` form prevents the NPE if `something` is a `nil` `interface{}`